key.js

```diff
@@ -1,277 +1,446 @@
-keyBoard.style.display = `none`;
-startButton.style.transform = `translateY(300px)`;
-const levelOne = () => {
-    lives++;
-    h3.display = `none`;
-    keyBoard.style.display = `none`;
-    startButton.style.transform = `translateY(200px)`;
-    startButton.style.fontSize = `100px`;
-}
+const controller = (function (game, view) {
 
-const homeAnimation = () => {
-    clearInterval(timer);
-    timerElement.style.display = `none`;
-    headings.innerText = `KEY FOCUS`
-    headings.style.color = ``;
-    startButton.style.fontSize = `100px`
-    startButton.textContent = `START`;
-    document.body.style.backgroundColor = ``;
-    h3.style.display = `none`;
-    startButton.style.transform = `translateY(300px)`
-    keyBoard.style.display = `none`;
-    startButton.style.color = ``;
-    startButton.style.backgroundColor = ``;
-    startButton.style.border = ``;
-    span.forEach(element => {
-        element.innerText = ``;
-    })
-}
+    function initGame() {
+        document.querySelector(view.DOMStrings.mainButton).addEventListener('click', () => {
+            view.animateToLevelZero();
+            game.gameStart();
+        })
+    }
 
-const startAnimation = () => {
-    timerElement.style.display = ``;
-    newSequence();
-    startButton.style.fontSize = `45px`
-    let seconds = 60;
-    timer = setInterval(() => {
-        seconds--;
-        timerElement.textContent = `${seconds} `;
-        if (!seconds) {
-            clearInterval(timer);
-        }
-    }, 1000)
-    timerElement.style.boxShadow = `0px 0px 4px 3px darkgray`;
-    startButton.style.transform = ``
-    h3.style.display = ``;
-    keyBoard.style.display = ``;
-    startButton.textContent = `HOME`;
-    startButton.style.transform = `translateX(600px)`
-    startButton.style.backgroundColor = `red`;
-    startButton.style.color = `white`;
-    document.body.style.backgroundColor = `whitesmoke`;
-    startButton.style.border = `5px solid darkred`
-    headings.textContent = `Baby Level`;
-    headings.style.color = "darkslategray";
-    levels();
-}
-startButton.addEventListener('click', () => {
-    buttonPress()
-    bool = !bool;
-    if (bool) {
-        startAnimation();
-    } else {
-        homeAnimation();
-    }
-})
+
+    const entireLogic = () => {
+        const arrowCount = document.querySelector(viewModule.DOMStrings.randomKeyArrows).length;
+        let index = 0;
+        // Key Interactions Logic : 
+        addEventListener('keyup', (e) => {
+            view.resetArrowStyles();
+            const keyPress = e.key;
+            const currentSpan = document.querySelector(viewModule.DOMStrings.randomKeyArrows)[index];
+            const currentSpanStyles = () => {
+                currentSpan.classList.add('color-gray', 'arrow-transition-to-gray', 'font-size-70');
+            }
+            const greenCurrentSpanColor = () => { currentSpan.style.color = 'green' }
+            const redCurrentSpanColor = () => { currentSpan.style.color = 'red' }
```

Comment:
Use currying like this:

```suggestion
            const checkCurrentArrowColor = color => () => currentSpan.style.color === color;
            const isArrowColorGreen = () => checkCurrentSpanStyleColor('green');
            const isArrowColorRed = () => checkCurrentSpanStyleColor('red');
```